README.md

```diff
@@ -26,6 +26,8 @@ make generate build
 
 ./bin/github_exporter -h
 ```
+Inorder to run it in debug mode (from an IDE: goland etc) , ensure `GITHUB_EXPORTER_DATABASE_DSN` is set to a
```

Comment:
The variable should have a useable default, so this should not be needed. To customize the used tags it should be part of the make command above as it can be provided like `make build TAGS="sqlite genji"`.

Reply:
Maybe it would be even better if the tags are not provided to default to some useful value depending on the OS and arch.

Reply:
As can be seen in ```func init() {
	if _, ok := store.Drivers["genji"]; ok {
		defaultDatabaseDSN = "genji://storage/exporter"
	} else if _, ok := store.Drivers["sqlite"]; ok {
		defaultDatabaseDSN = "sqlite://storage/exporter.sqlite3"
	}
}```, if the driver is `sqlite` we should point to `sqlite://storage/exporter.sqlite3` but when i was running it via an IDE (go land) with build, it would complain that the driver is not found. So , for me to be able to run it locally to contribute, I had to dig and find and set this variable to a local path. Maybe I am doing something wrong or we should have a step to how to run it in IDE for debugging and development.

Reply:
I never used Goland, how do you build the project? With the documented way from the README it should include SQLite and Genji by default.